Library/Source/NativeEngine.h

```diff
@@ -166,11 +130,17 @@ namespace babylon
         ~FrameBufferData()
         {
             bgfx::destroy(FrameBuffer);
-            m_idSet.Recycle(ViewId);
         }
 
-        void SetUpView()
+        void UseViewId(uint16_t viewId)
         {
+            ViewId = viewId;
+            ViewClearState.SetViewId(ViewId);
```

Comment:
If ViewClearState already has the ViewId, do we really need another one on this class?

Reply:
Yes. You can clear the backbuffer, set a new viewport, and call clear again with a different state.